src/api/emailApi.ts

```diff
@@ -8,37 +8,23 @@ import {
   VerifyEmailPayload,
 } from './types';
 
-export const subscribeEmail = (data: SubscribePayload) => {
-  api.post(`subscribe`, {
-    data,
-  });
-};
+export const subscribeEmail = (data: SubscribePayload) => api.post(`subscribe/`, { ...data });
```

Comment:
{ ...data } does nothing, can be written as `export const subscribeEmail = (data: SubscribePayload) => api.post(`subscribe/`, data);`
 comment relevant to all occurrences below